app/helpers/PermissionsHelpers.ts

```diff
@@ -0,0 +1,24 @@
+export enum PermissionStatus {
```

Comment:
less of a fan of `helper` files, especially for things that are domain concepts rather than utility functions.  What do you think about promoting this file to the top level as a named concept? something like:
`./app/permissionStatus.ts`